R/rd-html.R

```diff
@@ -204,13 +204,21 @@ as_html.tag_Sexpr <- function(x, ...) {
   on.exit(setwd(old_wd), add = TRUE)
 
   # Environment shared across a file
-  res <- eval(parse(text = code), context_get("sexpr_env"))
+  outlines <- utils::capture.output({
```

Comment:
Given that the capturing is only needed for verbatim, I think it would be slightly clearer to compute `results` earlier and then handle verbatim in it's own early-returning `if` block. Does that make sense?

Reply:
Makes sense. Done in the amended commit.